inference-engine/tests/functional/plugin/myriad/subgraph_tests/trivial_graphs.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <shared_test_classes/base/layer_test_utils.hpp>
```

Comment:
Can you enable these tests too?

- https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/constant_result.hpp
- https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/parameter_result.hpp

Reply:
I've enabled `parameter_result.hpp`, but unfortunately we still don't support `constant_result.hpp` since it has no parameters. I've also added `parameter_shapeof_result` to the shared part, so I've finally got rid of `trivial_graphs.cpp`.
Thanks!